deployments/daemon-set/nginx-plus-ingress.yaml

```diff
@@ -54,10 +64,20 @@ spec:
          #  memory: "1Gi"
         securityContext:
           allowPrivilegeEscalation: false
+          readOnlyRootFilesystem: false
```

Comment:
same here, set to `true` and comment

Reply:
As above, applied in [latest change](https://github.com/nginxinc/kubernetes-ingress/compare/bb17a8efaf8b2ed61daf2f9b44bd7f6839c74f38..a10301e4641bc272ec9d7af7978457340967b7eb). 🙏🏻